### PR TITLE
feat: [APAF-1452] qrcode element doc

### PR DESCRIPTION
**Pull request by chao-ding-awx**

### CODE REVIEW

File: docs/qrcode.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+```
+
+### 8. Add an `onError` event listener to handle events when the payment has failed.
+
+```js
+window.addEventListener('onError', (event) => {
+  /*
+    ... Handle event on error
+  */
+  console.log(event.detail);
```
Comment (olivia-wei-awx) [inline docs/qrcode.md:112]: why use `window.alert` above, but  use `console.log` here

File: integrations/react/src/components/Qrcode.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      const qrcode = createElement('qrcode', {
+        intent: {
+          // Required, must provide intent details to prepare qrcode element for checkout
+          id: intent_id,
+          client_secret,
+        },
+        qrcodeMethod: payment_method,
+      });
+      // STEP #5: Mount the element to the empty container created previously
+      qrcode.mount('qrcode'); // This 'qrcode' id MUST MATCH the id on your empty container created in Step 3
```
Comment (olivia-wei-awx) [inline integrations/react/src/components/Qrcode.jsx:42]: MUST MATCH why is  all uppercase

File: integrations/react/src/components/Qrcode.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+    window.addEventListener('onReady', onReady);
+    window.addEventListener('onSuccess', onSuccess);
+    window.addEventListener('onError', onError);
+    return () => {
+      window.removeEventListener('onReady', onReady);
+      window.removeEventListener('onSuccess', onSuccess);
+      window.removeEventListener('onError', onError);
+    };
+  }, []); // This effect should ONLY RUN ONCE as we do not want to reload Airwallex and remount the elements
```
Comment (olivia-wei-awx) [inline integrations/react/src/components/Qrcode.jsx:80]: ONLY RUN ONCE is alse all uppercase

File: integrations/react/src/components/Qrcode.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    width: '540px',
+    margin: '48px auto',
+  };
+
+  return (
+    <div>
+      <h2>Qrcode element integration</h2>
+      {/* Example below: show loading state */}
+      {!elementShow && <p>Loading...</p>}
+      {/* Example below: display response message block */}
```
Comment (olivia-wei-awx) [inline integrations/react/src/components/Qrcode.jsx:93]: message body my be better

File: integrations/react/src/components/Qrcode.jsx (hunk 2 above)
Comment (chao-ding-awx) [reply integrations/react/src/components/Qrcode.jsx:42]: I think want to make it more important
believe in @josie-ku-awx , she is professional in this.

File: integrations/react/src/components/Qrcode.jsx (hunk 3 above)
Comment (chao-ding-awx) [reply integrations/react/src/components/Qrcode.jsx:80]: believe in @josie-ku-awx , she is professional in this.

File: integrations/react/src/components/Qrcode.jsx (hunk 4 above)
Comment (chao-ding-awx) [reply integrations/react/src/components/Qrcode.jsx:93]: keep following the other components

File: docs/qrcode.md (hunk 1 above)
Comment (chao-ding-awx) [reply docs/qrcode.md:112]: yeah, changed the alert to console

**Review by olivia-wei-awx — APPROVED** [review]

**Pull request merged**